### PR TITLE
fix(map): Change styling of map in night mode

### LINKED ISSUE #15702: New navigation pane on beta has incorrect styling (night mode)
While doing a beta challenge, I discovered that a navigation pane (like a mini-map) is now available on the left. I think this is probably a recent addition.  

The navigation page's styling in night mode is way off - 

screenshot 2017-08-01 01 14 53

The same pane in the regular theme - 

screenshot 2017-08-01 01 16 25

I'm not sure if this has already been reported. I did a quick search, didn't come up with anything.

### PR DESCRIPTION
**Pull request by markusenglund**
Previously the map had white background color and white color, making it
impossible to see the text. I changed the colors to the standard
night-mode color variables. I also deleted a css class that was not
doing anything.

To QA this, you have to be in night mode, however since the night mode toggle button doesn't currently work you might have to do a workaround like adding the following to local storage:

key: fcc-theme
value: "night"

Closes #15702

#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #15702

### CODE REVIEW

**Review by systimotic — CHANGES_REQUESTED** [review]
I'm sorry you didn't hear from anybody in so long @yogaboll. 
This is a great improvement 👍
I only found one thing. The carets (those arrows that show whether a block is open or closed, class .map-caret) are still black, so you can't see them. Can you change them to the text color as well? Thanks!

**Comment by yogaboll** [comment]
Requested changes have been made.

File: common/app/Map/map.less [hunk 1]
```diff
@@ -121,7 +121,13 @@
   .@{ns}-accordion a:focus, #noneFound {
     color: #ABABAB;
   }
-  .challenge-title {
+  .@{ns}-challenge-title, .@{ns}-accordion-panel-heading {
+    background: @night-body-bg;
```
Comment (systimotic) [inline common/app/Map/map.less]: The text color will need to be added here too. The circle icons next to the challenge names are green, while they should be gray.

**Review by systimotic — CHANGES_REQUESTED** [review]
Update is excellent 👌 

I noticed one more thing I missed last time, so sorry! I always think it's annoying to get feedback on something that was the same way the last time that person gave feedback on it, yet I often find myself doing it too 😅

File: common/app/Map/map.less (hunk 1 above)
Comment (yogaboll) [reply common/app/Map/map.less]: Request and you shall receive.

Comment (yogaboll) [reply common/app/Map/map.less]: Circle icons are now grey.

**Review by systimotic — APPROVED** [review]
## LGTM 🎉 🎉 
Thanks for the fix @yogaboll! Nice work on the updates too. Looking forward to seeing more of your contributions

**Pull request merged**